tests/flow/test_defrag.py

```
@@ -496,3 +496,105 @@ def churn():
+            if finished and state["error"] is None:
+                res = self.db.select_graph(gname).query(
+                    "MATCH (n:Node) RETURN count(n)").result_set
+                self.env.assertEquals(res[0][0], BATCHES * BATCH)
+
+        finally:
+            t.join(timeout=5)
+            for k, v in original_cfg.items():
+                try:
+                    conn.config_set(k, v)
+                except Exception:  # noqa: BLE001
+                    pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)